src/kit/LogViewer/LogViewerSelect.tsx

```diff
@@ -83,7 +87,10 @@ const LogViewerSelect: React.FC<Props> = ({
     for (let i = 0; i < keys.length; i++) {
       const key = keys[i] as keyof Filters;
       const value = values[key];
-      if (value && value.length !== 0) return true;
+      if (key === 'enableRegex' && value) {
```

Comment:
q: so it seems like there's currently no way to toggle `enableRegex` other than manually changing settings, right?